recipes/progress_reporter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "progress_reporter" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }} 
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/marscher/progress_reporter
+  git_rev: {{ 'v' + version }}
+
+build:
+  noarch_python: True
+  number: 0
+# since this is noarch, only build on linux
+  skip: True  # [not linux]
+  script: python setup.py install
```

Comment:
What does that mean? I dont have any dependencies which setuptools could download during the build process. So think I dont need these special args. Can you tell me why I should use them?